Week2/homework/js-exercises/removeComma.js

```diff
@@ -0,0 +1,7 @@
+'use script';
+{
+    let myString = 'hello,this,is,a,difficult,to,read,sentence';
+    console.log(myString.length);
+    myString = myString.replace(/,/g, " ");
```

Comment:
👍 

You can also achieve the same using `split()` and `join()` methods on `String`, you can loop up MDN to see how to use them!